localization/english/dvg_flavor_venice_l_english.yml

```diff
@@ -1,27 +1,162 @@
-venice_background.1.a:0 "I want to know more of Venice's history."
-venice_background.1.b:0 "I want to know more about Venice's possessions."
-venice_background.1.c:0 "I want to know more about the Patrician's and Venice's political situation."
-venice_background.1.d:0 "Desponsamus te, mare, in signum veri perpetuique dominii."
+# venice.1.t:0 "The History and Present Situation of Venice"
+# venice.1.d:0 "The Republic of Venice stands on the crossroads of the Mediterranean. Queen of the Adriatic, Venice's power, wealth, and status have greatly diminished since its heights during the Renaissance. Though still in possession of a vast colonial empire stretching from Cyprus to Athesia, the recent intervention in the Ottoman Civil War and acquisition of the Greek mainland has become an evergrowing headache. Greek nationalists increasingly call for a fully independent Greece, while to the West, nationalistic forces are plotting to unify Italy, once and for all. Yet in the face of these threats, the Patrician nobility continues to resist the winds of change, fearful that any change to the status quo will end their vicelike grip over power. As a Doge, you will have to guide Venice during this perilous century or watch as Venice fades into ever greater irrelevance."
+# venice.1.f:0 "To build a city where it is impossible to build a city is madness in itself, but to build there one of the most elegant and grandest of cities is the madness of genius. - Alexander Herzen"
+# venice_background.1.a:0 "I want to know more of Venice's history."
+# venice_background.1.b:0 "I want to know more about Venice's possessions."
+# venice_background.1.c:0 "I want to know more about the Patrician's and Venice's political situation."
+# venice_background.1.d:0 "Desponsamus te, mare, in signum veri perpetuique dominii."
+
+dvg_ven.1.t:0 "The History and Situation of Venice"
+dvg_ven.1.d:0 "The Winds of Change Are Rising"
+dvg_ven.1.f:0 "The Republic of Venice stands on the crossroads of the Mediterranean. Queen of the Adriatic, Venice's power, wealth, and status have greatly diminished since its heights during the Renaissance. Though still in possession of a vast colonial empire stretching from Cyprus to Athesia, the recent intervention in the Ottoman Civil War and acquisition of the Greek mainland has become an evergrowing headache. Greek nationalists increasingly call for a fully independent Greece, while to the West, nationalistic forces are plotting to unify Italy, once and for all. Yet in the face of these threats, the Patrician nobility continues to resist the winds of change, fearful that any change to the status quo will end their vicelike grip over power. As a Doge, you will have to guide Venice during this perilous century or watch as Venice fades into ever greater irrelevance."
```

Comment:
Make the nation name hoverable/link. You can also do that with cultures and maybe also IG's. Makes the event more interactive.